README.md

```diff
@@ -69,6 +70,33 @@ jobs:
     - name: Cache workaround # https://github.com/actions/cache/issues/133
       run: chmod -R a+rwx .
 ```
+You can also use the provided Github Action:
+```
+name: GitHub Action sample
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build_test_and_deploy:
+    runs-on: ubuntu-18.04 # Works also with self hosted runner supporting docker
+    container:
+        image: docker://docker:stable-git
+
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v2.1.0
+
+    - name: Test action
+      uses: fabernovel/docker-android
```

Comment:
The action is not published yet. Github doesn't seems to recognise this repository as an action holder. We might need to move everything out of the action directory. (or create another repository) 

See #55

Reply:
We can move everything out action directory, as in action.yml we can define which Dockerfile to use. We will adresse that with #55.

Reply:
Will do